CHANGES

```diff
@@ -33,6 +33,20 @@
 # New collective algorithms. All collective algorithms are listed in
   src/mpi/coll/coll_algorithms.txt
 
+# Removed hydra2.
+
+# Added in-repository wiki documentations.
+
+# Added stream workq to support optimizations for enqueue operations.
+
+# Better support for large count APIs by eliminating type conversion issues.
+
+# PMI refactor.
+
+# Hydra refactor.
```

Comment:
I'm not sure these are notable on their own. Maybe we can call out specific new capabilities?